src/note-components/authentication/login.jsx

```diff
@@ -0,0 +1,99 @@
+import { Navbar } from "../navbar/navbar";
+import "./auth.css";
+import { useAuth } from "../../Context/authorization-context";
+import { useState } from "react";
+import { loginAPI } from "../../note-API/auth-api";
+import { Link, useNavigate } from "react-router-dom";
+
+
+
+const Login = () => {
+const [ user, setUser ] = useState({email : "", password : ""});
+const { authState, authDispatch } = useAuth();
+const navigate = useNavigate();
+
+const guestUserInfo = {
+email: "adarshbalika@gmail.com",
+password: "adarshBalika123",
+};
+
+const newInfoHandler = (event) => {
+const { id, value } = event.target;
+setUser({ ...user, [id]: value });
+};
+
+const guestUserHandler = (event) => {
+event.preventDefault();
+setUser(guestUserInfo);
+};
+
+const loginHandler = async (event) => {
+event.preventDefault();
+if ((user.email !== "", user.password !== "")) {
```

Comment:
Suggestion:  Instead of checking here, you can add required attribute in the email and password field.